test-requirements.txt

```diff
@@ -10,7 +10,7 @@ matplotlib
 mongoquery
 networkx
 nose
-ophyd
+git+git://github.com/tacaswell/ophyd.git@enh_pseudopos_extensions#egg=ophyd
```

Comment:
Do you plan to blow this commit away before merging? I would rather take your word that this combination of branches pass or, at most, aim this as ophyd master.

Reply:
Yes, the ophyd one needs to go in first.  I want it to run on travis so I can look at the coverage.